docs/src/tutorials/creating_geometry.md

```diff
@@ -0,0 +1,251 @@
+# Creating Geometry
+
+````@example creating_geometry
+import GeoInterface as GI
+import GeometryOps as GO
+import GeoFormatTypes as GFT
+import CoordinateTransformations
+import Proj
+using CairoMakie
+using GeoMakie
+using GeoJSON
+````
+
+The first thing we need to do is decide which Coordinate Reference System (CRS) we will be working in. Here, we start with the most common geographic CRS (i.e. coordiantes of latitude and longitude), [WGS84](https://epsg.io/4326).
+
+````@example creating_geometry
+crs = GFT.EPSG(4326)
+````
+
+Let's start by making a single `Point` with CRS info included.
+
+````@example creating_geometry
+point = GI.Point(0, 0; crs)
```

Comment:
Would really like to discourage people from doing this with points... Crs on the wrapper is always enough

Reply:
I think this should be addressed with a Note and not a change. As we ease users into how to great geospatial geometries a point with crs is the simplest geometry that can be created. But we do need a note on how the CRS is interpreted and that is only needs to be included in the highest level wrapper.

Reply:
A point without crs is the simplest!!

The idea that points need CRS will stick if it's the first learning experience.

Reply:
It has terrible consequences, like you can't `reinterpret` the vector as tuples, and you use 1.5 - 2x the memory.

Reply:
If I wanted to plot a single point on a map, and have the CRS info interpreted, is this not how I would construct it?

Reply:
But that's an edge case that basically no-one does.

Creating a Point to put in something is the 99% use case, and we should teach those, not the 1%